work-with-us/index.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Work With Us
+sub_title: Effective Design, Quality Code
+---
+
+<section>
+  <div class='container typography'>
+    <h1>Work with us</h1>
+
+    <p>Not ready to <a href='mailto:contact@apsis.io'>contact us</a> for an estimate? Here are some of the best reasons why you should choose Apsis Labs.</p>
+
+    <h2>Grow Your Business</h2>
+
+    <p>Do you need sensible software development? We might be a great for you if you:</p>
+
+    <ul>
+      <li>Have a great app idea for your business or soon-to-be business, but you’re not sure how to turn the idea into a reality.</li>
+      <li>Spend hours of your valuable time moving data between spreadsheets or documents to track your business information.</li>
+      <li>Hired your own software engineers, but your backlog is full of killer features you'll never have time to implement.</li>
+      <li>Want to build an app or website that handles sensitive data, but you aren't sure how to make sure it's technically compliant with HIPAA.</li>
+      <li>Expend hours doing repetitive work that is important to you, but is not a core part of your business.</li>
+    </ul>
+
+    <h2>Whole Business Approach</h2>
+
+    <p>We know your business is unique and complex. That’s why we:</p>
+
+    <ul>
+      <li>Make sure that we thoroughly understand your business, your vision and your existing processes, before we ever write a line of code.</li>
```

Comment:
I'm not a grammar person but the commas feel wrong here.

Reply:
Maybe just: `Make sure that we thoroughly understand your business, vision, and existing processes before we ever write a line of code`